react/src/lib/components/VectorCalculator/components/ExpressionInputTextField.tsx

```diff
@@ -24,42 +29,38 @@ export const ExpressionInputTextField: React.FC<ExpressionInputTextFieldProps> =
     (props: ExpressionInputTextFieldProps) => {
         const { expression, status, helperText, disabled } = props;
 
-        const [textFieldVariantState, setTextFieldVariantState] =
-            React.useState<ExpressionInputTextFieldVariantType>("default");
-        const [textFieldIconState, setTextFieldIconState] =
-            React.useState<React.ReactNode | undefined>(undefined);
+        const [textFieldAnimationDataState, setTextFieldAnimationDataState] =
+            React.useState<ExpressionInputTextFieldAnimationData>({
```

Comment:
Is `Animation` the correct term? AFAIK nothing is moving/animated. Maybe `StyleData`?